evcache-core/src/main/java/com/netflix/evcache/util/KeyHasher.java

```diff
@@ -45,40 +65,40 @@
 //        }
 //    }
 
-    public static String getHashedKey(String key, String hashingAlgorithm) {
+    public static String getHashedKey(String key, HashingAlgorithm hashingAlgorithm) {
```

Comment:
Let us also add an overloaded method like max hashing bytes. 
For ex, I want to use murmur3 but only first 12 bytes instead of 16 bytes.

Reply:
done